src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.spec.ts

```diff
@@ -22,6 +27,18 @@ describe('DeploymentsDonutChartComponent', () => {
     pods = Observable.of({ pods: [['Running', 1], ['Terminating', 1]], total: 2 });
     desiredReplicas = 1;
     isIdled = false;
+    colors = {
```

Comment:
Please use the PatternFly colors for all of these items and comment what variables they are. Example:
```
'Running': '#004368', // @color-pf-blue-600
```